src/order/filter/basic.lean

```diff
@@ -274,14 +274,17 @@ def join (f : filter (filter α)) : filter α :=
 end join
 
 section lattice
+variables {f g : filter α} {s t : set α}
```

Comment:
Again, I'd lean towards not adding these in this PR, it make the section below inconsistent.

Reply:
Given that I'm using thos efive times already, I will keep them here.